### PR TITLE
CSW - Fix CUP CSW

### PR DESCRIPTION
**Pull request by johnb432**
**When merged this pull request will:**
- With only CBA, ACE and CUP Weapons loaded, the existing CSW config does not work.
I didn't try if it worked if you have CUP Units and/or Vehicles loaded, as I made this fix independently from that.

### IMPORTANT

- If the contribution affects [the documentation](https://github.com/acemod/ACE3/tree/master/docs), please include your changes in this pull request so the documentation will appear on the [website](https://ace3.acemod.org/).
- [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

**Review by Mike-MF — COMMENTED** [review]
I'm probably the worst person to review this, I have no idea what CSW is or how it works.

Although I now realise this should probably depend on Vehicles as the `ace_csw_mortarBaseplate` entries do come from there.

File: addons/compat_cup_weapons/compat_cup_weapons_csw/CfgWeapons.hpp [hunk 1]
```diff
@@ -104,8 +104,8 @@ class CfgWeapons {
             deployTime = 4;
             pickupTime = 4;
             class assembleTo {
-                ace_csw_kordTripod = "CUP_O_KORD_high_RU";
-                ace_csw_kordTripodLow = "CUP_O_KORD_RU";
+                ace_csw_kordTripod = "CUP_I_KORD_AAF";
+                ace_csw_kordTripodLow = "CUP_I_KORD_high_AAF";
```
Comment (TheCandianVendingMachine) [inline addons/compat_cup_weapons/compat_cup_weapons_csw/CfgWeapons.hpp:108]: swap these

**Review by TheCandianVendingMachine — CHANGES_REQUESTED** [review]

File: addons/compat_cup_weapons/compat_cup_weapons_csw/CfgWeapons.hpp (hunk 1 above)
Comment (TheCandianVendingMachine) [reply addons/compat_cup_weapons/compat_cup_weapons_csw/CfgWeapons.hpp:108]: low tripod doesnt make high weapon, for example

Comment (johnb432) [inline addons/compat_cup_weapons/compat_cup_weapons_csw/CfgWeapons.hpp:108]:
```suggestion
                ace_csw_kordTripod = "CUP_I_KORD_high_AAF";
                ace_csw_kordTripodLow = "CUP_I_KORD_AAF";
```

Comment (johnb432) [reply addons/compat_cup_weapons/compat_cup_weapons_csw/CfgWeapons.hpp]: Done.

**Review by TheCandianVendingMachine — APPROVED** [review]
as long as the weapons are real static weapons, this lgtm

**Review by Mike-MF — APPROVED** [review]
All the classes exist

**Pull request merged**